src/app/shutter/shutter.component.html

```diff
@@ -4,7 +4,7 @@
 <div class="container-fluid">
   <div class="block-wrapper">
     <div class="block mt-xl">
-      <h2 class="heading-h2">This application is not available for your jurisdiction(s) on this address any more. You're being redirected to its new address, where it will be better supported with more features to come in the future.</h2>
+      <h2 class="heading-h2">This application has moved to a new URL. You'll automatically be redirected. <a href="{{getRedirectUrl()}}">View the new application</a> if you're not redirected in 10 seconds</h2>
```

Comment:
full stop at end of the sentence, please.

Reply:
Done